plotFADC/FADCtest.nim

```diff
@@ -0,0 +1,180 @@
+import nimhdf5, os, ingrid/tos_helpers, sequtils, strutils
+import sets
+import plotly
+import tables
+import re
+import docopt
+
+
+when defined(linux):
+  const commitHash = staticExec("git rev-parse --short HEAD")
+  const currentDate = staticExec("date")
+else:
+  const commitHash = ""
+  const currentDate = ""
+
+const docTmpl = """
+InGrid/FADC reading and cuting.
+
+Usage:
+ FADCtest <HDF5file> [options]
+ FADCtest <HDF5file> --runNumber <runnum> [options]
+ FADCtest <HDF5file> --runNumber <runnum> --chipNumber <chipnum> [options]
+ FADCtest <HDF5file> --chipNumber <chipnum> [options]
+
+Options:
+ -h --help show this text
+ --runNumber      choose the run you would like to look at
+ --chipNumber     choose the chip you would like to look at
+ if not selected the program chooses the last values it gets from the file
+
+ """
+
+const doc = docTmpl % [commitHash, currentDate]
+
+proc readandcut*(h5f: var H5FileObj, runNumber: int, chip: int) =
```

Comment:
Have argument about which datasets to use: FADC dataset (or should that always be the same anyways) and against which dataset to compare.